package.json

```
@@ -90,7 +90,7 @@
     "start:msw:stage": "NODE_ENV=development MSW=TRUE fec dev --clouddotEnv=stage",
     "test": "TZ=UTC vitest run",
     "test:watch": "TZ=UTC vitest",
-    "test:coverage": "TZ=UTC vitest run --coverage",
+    "test:coverage":  "TZ=UTC vitest run --coverage --reporter=junit --outputFile=./coverage/junit.xml",
```

Looks like there is one extra space, but otherwise looks good to me!

```suggestion
    "test:coverage": "TZ=UTC vitest run --coverage --reporter=junit --outputFile=./coverage/junit.xml",
```